contentScript.js

```diff
@@ -526,9 +598,12 @@ const bindSubmitButtonToSaveInfo = () => {
     window.localStorage.setItem("searchpref", searchPreftext);
     window.localStorage.setItem("pincode", first_5_pin_digits);
     window.localStorage.setItem("timeslot", timeslotind);
+    window.localStorage.setItem("costpreference", costPreferenceValue);
+    window.localStorage.setItem("vaccinepreference", vaccinePreferenceValue)
     window.localStorage.setItem("centerprefs", center_prefs_string);
     window.localStorage.setItem("minavailability", minavailability);
     window.localStorage.setItem("autoconfirm", enableautoconfirm);
+    window.localStorage.setItem("selectedbuttoncheckboxes", selected_button_checkbox)
```

Comment:
does setItem accept an array?

Reply:
setItem converts array into comma separated strings. I will use JSON.stringify to store the array, that seems better

Reply:
Yeah, that would be great, un-typed bugs should be avoided first hand!